exchanges/exchange.go

```diff
@@ -222,16 +223,16 @@ func (e *Base) GetPairAssetType(c currency.Pair) (asset.Item, error) {
 
 // GetClientBankAccounts returns banking details associated with
 // a client for withdrawal purposes
-func (e *Base) GetClientBankAccounts(exchangeName, withdrawalCurrency string) (config.BankAccount, error) {
+func (e *Base) GetClientBankAccounts(exchangeName, withdrawalCurrency string) (banking.Account, error) {
```

Comment:
return pointer like below?

Reply:
Was considering moving these out of config into banking but left them there because you are updating config values at this point but made change to return pointer as we are dealing with a slice behind the scenes

Reply:
👍